### PR TITLE
feat(vx-brush): adds resetOnEnd property

### PR DESCRIPTION
**Pull request by gillesdemey**
This will reset the brush after dragging, we've overlay-ed the `Brush` element on top of a chart so we can have zoom interaction without an additional smaller brushable chart.

brush-zoom-reset

#### :boom: Breaking Changes

- none

#### :rocket: Enhancements

- add a `resetOnEnd` property to the `BaseBrush` and `Brush` component.

#### :memo: Documentation

- No documentation yet for the `Brush` component :)

#### :bug: Bug Fix

- none

#### :house: Internal

- none

### CODE REVIEW

**Comment by gillesdemey** [comment]
Not sure why the test suite is failing, but at first glance seems unrelated to the `@vx/brush` component 🤔

File: packages/vx-brush/src/BaseBrush.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export type BaseBrushProps = {
   onClick?: (event: MouseHandlerEvent) => void;
   clickSensitivity: number;
   disableDraggingSelection: boolean;
+  resetOnEnd: boolean;
```
Comment (williaster) [inline packages/vx-brush/src/BaseBrush.tsx]: one nit – can we make this optional? `resetOnEnd?`

Comment (gillesdemey) [reply packages/vx-brush/src/BaseBrush.tsx]: Yep absolutely, should have thought of that 😅

**Comment by williaster** [comment]
@gillesdemey could you try rebasing this on the latest master to see if it fixes the build? there have been PRs opened since this that have passed CI.

**Comment by williaster** [comment]
#616 which duplicates this on the latest `master` does seem to work for me.

**Comment by gillesdemey** [comment]
Looks like CI is green 🎉

**Comment by williaster** [comment]
amazing, thanks again for the add, should be out soon in `0.0.194`

**Pull request merged**